iset.mm

```diff
@@ -116197,6 +116276,361 @@ Infinite sums (cont.)
   $}
 
 
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+  Mertens' theorem
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+$)
+
+  ${
+    $d G a k n $.  $d G k n z $.  $d S a k n $.  $d S k n z $.  $d X a n $.
+    $d X n z $.  $d a k n ph $.
+    mertenslemub.gb $e |- ( ( ph /\ k e. NN0 ) -> ( G ` k ) = B ) $.
+    mertenslemub.b $e |- ( ( ph /\ k e. NN0 ) -> B e. CC ) $.
+    mertenslemub.cvg $e |- ( ph -> seq 0 ( + , G ) e. dom ~~> ) $.
+    mertenslemub.t $e |- T = { z | E. n e. ( 0 ... ( S - 1 ) ) z =
+                        ( abs ` sum_ k e. ( ZZ>= ` ( n + 1 ) ) ( G ` k ) ) } $.
+    mertenslemub.elt $e |- ( ph -> X e. T ) $.
+    mertenslemub.s $e |- ( ph -> S e. NN ) $.
+    $( Lemma for ~ mertensabs .  An upper bound for ` T ` .  (Contributed by
```

Comment:
"upper bound fo ` X `"?

Reply:
Since `X` is an arbitrary element of `T` (and the bound doesn't depend on `X`), it is an upper bound for `T`.